test/Microsoft.AspNetCore.Razor.Test/project.json

```diff
@@ -3,6 +3,7 @@
   "dependencies": {
     "dotnet-test-xunit": "2.2.0-*",
     "Microsoft.AspNetCore.Razor": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Runtime": "1.1.0-*",
```

Comment:
CaseSensitiveTagHelperAttributeComparer needs TagHelperAttribute and NullHtmlEncoder which are in Razor.Runtime. Is there another way to reference them?